client/src/pages/ProfileDetail/BookingForm/BookingForm.tsx

```diff
@@ -0,0 +1,125 @@
+import useStyles from './useStyles';
+import { Box, Typography, Select, MenuItem, OutlinedInput, Grid, Button, Card } from '@material-ui/core';
```

Comment:
You aren't using the right material UI version here, so please update this to MUI v5.